packages/cli/src/commands/deploy/latest.ts

```diff
@@ -53,10 +53,12 @@ import validatePaths, {
   validateRootDirectory,
 } from '../../util/validate-paths';
 import { readLocalConfig } from '../../util/config/files';
-import { getCommandName } from '../../util/pkg-name.ts';
-import { getPreferredPreviewURL } from '../../util/deploy/get-preferred-preview-url.ts';
+import { getCommandName } from '../../util/pkg-name';
+import { getPreferredPreviewURL } from '../../util/deploy/get-preferred-preview-url';
+import { Output } from '../../util/output';
+import Client from '../../util/client';
 
-const addProcessEnv = async (log, env) => {
+const addProcessEnv = async (log: (str: string) => void, env: any) => {
```

Comment:
@styfle The `env` parameter it is complicated of type because it changes type during code execution 🤔

Reply:
You can probably use `typeof process.env` instead of `any`.

Reply:
Thank You, @TooTallNate ! `typeof process.env` I was the correct type 👍